site/content/guides/monitor-many-urls.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Monitor 500 pages in 2 minutes with Checkly
+displayTitle: Monitor 500 pages in 2 minutes with Checkly
+description: >-
+  By leveraging arrays, sitemap parsing, and dynamic monitor creation, you can ensure comprehensive coverage of your website, API routes, and third-party services—all from a single configuration file.
+author: Nočnica Mellifera
+avatar: 'images/avatars/nica-mellifera.png'
+tags:
+  - FAQ
+---
+
+Checkly’s model of [Monitoring as Code](https://www.checklyhq.com/guides/getting-started-with-monitoring-as-code/) lets you create site monitors right from your IDE. With the recent release of uptime monitoring, we have the ability to monitor every single URL, API route, and even third party service that makes up our product. We don’t want to write out dozens or hundreds of files for all this monitoring coverage. 
+
+If this is your first interaction with Monitoring as Code, check out our guide on [creating URL Monitors with Monitoring as Code](https://www.checklyhq.com/guides/url-monitoring/). That guide is all you need to create a project with a first monitor.
+
+## tl;dr Here’s how to create lots of monitors at once
+
+Here’s a single file that will create URL monitors from an array:
+
+```ts
+//urlArrayMonitors.check.ts
+//this will create URL monitors in your Checkly dashboard
+
+import { Frequency, UrlMonitor, UrlAssertionBuilder } from 'checkly/constructs'
+
+//just two URLs in this example, add as many as you want
+const sitemapUrls = [
+    'https://docs.anthropic.com/de/api/admin-api/workspace_members/get-workspace-member',
+    'https://docs.anthropic.com/de/api/admin-api/workspace_members/list-workspace-members'
+]
+
+//create paths and friendly names for each monitor
+sitemapUrls.forEach((url, index) => {
+  const urlPath = new URL(url).pathname.replace(/\//g, '-').replace(/^-+|-+$/g, '') || 'root'
+  const monitorId = `checkly-${urlPath}-ping`
+  const monitorName = `${urlPath.replace(/-/g, ' ')} pinger`
+
+//create each monitor with a five minute interval
+  new UrlMonitor(monitorId, {
+    frequency: Frequency.EVERY_5M,
+    name: monitorName,
+    activated: true,
+    request: {
+      url: url,
+      skipSSL: false,
+      followRedirects: true,
+      assertions: [
+        UrlAssertionBuilder.statusCode().equals(200),
+      ]
+    }
+  })
+})
+```
+
+This takes an array of URLs and creates a new monitor with a unique ID and name. We can test run these tests with `npx checkly test` and both monitors will run in our CLI’s default region.
```

Comment:
Maybe I'm missing something but its unclear to me what to do with this file. Where does it live? Where do the generated monitors land? How can I execute it?